ftw/simplelayout/aliasblock/__init__.py

```diff
@@ -0,0 +1,4 @@
+def initialize(context):
```

Comment:
@busykoala You sure this is required, see my comment above.

Reply:
This is required for the installation of the profile. The test will fail because the layer `ftw.testing.genericsetup.ZCMLLayer` can't be setup anymore. Also see the mapblock profile implementation as a reference.